test/test_nn.py

```diff
@@ -13399,6 +13399,136 @@ def test_linear_state(self):
         with self.assertRaisesRegex(RuntimeError, 'shape of an uninitialized'):
             module.load_state_dict(lazy_module.state_dict())
 
+    def check_lazy_conv(self, cls, lazy_cls, func, init_args, input_shape, expected_weight_shape):
```

Comment:
nit: you can make these helper functions start with an `_` to show that they are internal.

Reply:
Fixed!